src/editor/block/block.js

```diff
@@ -8,7 +8,10 @@ import { heynoteEvent, LANGUAGE_CHANGE } from "../annotation.js";
 import { SelectionChangeEvent } from "../event.js"
 import { mathBlock } from "./math.js"
 import { emptyBlockSelected } from "./select-all.js";
+import { newUpdatedTime, displayTime, timeMatcher } from "../time.js";
+import { LANGUAGES } from '../languages.js';
 
+const languageTokensMatcher = LANGUAGES.map(l => l.token).join("|")
```

Comment:
I had to copy this snippet over a few times - perhaps this goes in `languges.js` and can be imported?